horace_core/sqw/PixelData/@PixelDataFileBacked/private/activate_.m

```diff
@@ -0,0 +1,36 @@
+function [obj,is_tmp] = activate_(obj,filename,no_tmp_file_setting)
```

Comment:
I still don't get the point of `activate`/`deactivate` having read this. Why not a `move` which closes, moves, reopens? Why do we need to expose this functionality in parts?

Reply:
well, as I've said, activate/deactivate currently used for two purposes: move file to new place/name & distributing info about opened files to parallel workers (this was in `faccessor-s` - here I've just migrated the design. 
May be its not the best design but the best I was able to come within current time frames.
I wroted bunch of unit tests which check various user cases with file motion.  Better design which does that but simple is welcome.